Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -79,6 +79,10 @@ public class UserManager : IUserManager
         private IAuthenticationProvider[] _authenticationProviders;
         private DefaultAuthenticationProvider _defaultAuthenticationProvider;
 
+        private IPasswordResetProvider[] _passwordResetProviders;
+        private DefaultPasswordResetProvider _defaultPasswordResetProvider;
+        private Dictionary<string, IPasswordResetProvider> _activeResets = new Dictionary<string, IPasswordResetProvider>();
```

Comment:
Shouldn't this be a ConcurrentDictionary?

Reply:
Probably yes, but its technically meaningless at the moment. I was working on ways to better support and secure per user but I would have had to make enough changes that would impact the API that it didn't seem immediately worth it. Basically that entire dictionary can be removed right now with no ill effect.